tests/plugins/dialogadvtab/manual/multiplelinks.md

```diff
@@ -0,0 +1,25 @@
+@bender-tags: 4.10.1, 1046, bug
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, link, dialogadvtab, sourcearea
+
+1. Select "foo" word.
+1. Press "Link" button.
+1. Set URL to "aa".
+1. Go to "Advanced" tab.
+1. Set Id filed to "aa".
+1. Click OK.
+1. Put the selection in the "bar" link.
```

Comment:
TBO I find `Repeat above steps for "bar" word` easier to track, so I know that there are no strings attached to the steps and I could just repeat it without thinking. It may simplify this test. WDYT?

Reply:
I wasn't happy about this TC even at the time of writing, as I saw there are few too many steps. The point was to keep it dead simple, so that one could just follow step by step with explicit instructions.

But I think that referencing initial steps will improve the readability, and at the same time will be very easy to follow.